addons/website_sale/tests/test_customize.py

```diff
@@ -7,10 +7,10 @@
 from odoo.tools.misc import file_open
 
 from odoo.addons.base.tests.common import HttpCaseWithUserDemo, HttpCaseWithUserPortal
-
+from odoo.addons.sale.tests.product_configurator_common import TestProductConfiguratorCommon
 
 @tagged('post_install', '-at_install')
-class TestCustomize(HttpCaseWithUserDemo, HttpCaseWithUserPortal):
+class TestCustomize(HttpCaseWithUserDemo, HttpCaseWithUserPortal, TestProductConfiguratorCommon):
```

Comment:
Mhhhhhhhm, the ProductConfiguratorCommon is doing way more than what you need.  Is it really needed to make that change ?

Reply:
The test I added in that file needs `TestProductConfiguratorCommon`, and the second test I changed in the same file was defining the same product that was already defined in `TestProductConfiguratorCommon`. I could just have created a new class and don't change anything here but I thought it was better to factorize that here :)